apps/frontend/src/app/5_pages/ConvertPage/ConvertPage.tsx

```diff
@@ -282,10 +317,13 @@ const ConvertPage: FC = () => {
     ],
   );
 
-  const renderDestinationAmount = useMemo(
-    () => quote || t(commonTranslations.na),
-    [quote],
-  );
+  const renderDestinationAmount = useMemo(() => {
+    if (sourceToken === SupportedTokens.mynt) {
+      return convertedMyntAmount || t(commonTranslations.na);
+    } else {
+      return quote || t(commonTranslations.na);
+    }
+  }, [quote, convertedMyntAmount, sourceToken]);
```

Comment:
amounts are retrieved from router
```suggestion
  const renderDestinationAmount = useMemo(
    () => quote || t(commonTranslations.na),
    [quote],
  );
```